controllers/mattermost/mattermost/mattermost_test.go

```diff
@@ -292,6 +295,7 @@ func TestCheckMattermost(t *testing.T) {
 		}
 		err = reconciler.Client.Update(context.TODO(), job)
 		require.NoError(t, err)
+		assert.False(t, recStatus.ResourcesReady)
```

Comment:
Here we are not checking anything as we already checked this `recStatus`. This check should be after next call to `checkMattermostDeployment` and we should expect `True`.

Reply:
okay